source/process/community-guidelines.rst

```diff
@@ -4,11 +4,43 @@ Mattermost Community Guidelines
 
 This document provides easy-to-follow community guidelines for:
 
-- `GitHub Help Wanted Issues in Platform Repository`_
-- `Mattermost Community Forums`_
+.. contents::
+  :backlinks: top
+  :local:
+  :depth: 2
 
 ----
 
+Mattermost Community Playbook
+---------------------------------------------------------
+
+How to get community involved in a campaign?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Examples include React Native Apps, Redux and APIv4 campaigns.
+
+On-boarding
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Create a forum post letting developers know about the campaign and ways to contribute
+2. Create a contribution guide including developer machine set up
+3. Create help wanted tickets that a community member could work on
+ 
+ - For a high priority Help Wanted ticket, reach out to contributors directly in the GitHub repository for best engagement
+
+Retention
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Identify active contributors and thank them personally for their work
+2. For work in progress tickets or submitted pull requests, follow up with contributor once a week for status updates and questions. If there hasn't been activity in more than two weeks, ask if you can help them in some way
```

Comment:
maybe phrase this around helping make sure contributors aren't blocked with questions